lib/letter_avatar/avatar.rb

```diff
@@ -31,6 +31,10 @@ def cache_path
         "#{LetterAvatar.cache_base_path || 'public/system'}/letter_avatars/#{VERSION}"
       end
 
+      def font
+        "#{LetterAvatar.font || 'Roboto-Medium'}"
+      end
```

Comment:
also, please remove this method.. IMHO it is ok just to set this Roboto-Medium directly in `LetterAvatar.font`